src/vos/tests/vos_tests.c

```diff
@@ -116,13 +116,13 @@ run_all_tests(int keys, bool nest_iterators)
 	dts_create_config(cfg_desc_io, "keys=%d bypass=%s iterator=%s", keys,
 		      bypass, it);
 
-	for (i = 0; dkey_feats[i] >= 0; i++) {
-		for (j = 0; akey_feats[j] >= 0; j++) {
-			feats = dkey_feats[i] | akey_feats[j];
-			failed += run_io_test(feats, keys, nest_iterators,
-					      cfg_desc_io);
-		}
-	}
+	for (i = 0; dkey_feats[i] >= 0; i++) { //DM
```

Comment:
(style)  do not use C99 // comments
(style)  trailing statements should be on next line